### ISSUE TITLE
[Documentation] Clarification between the differences of a) Oracle GraalVM at https://www.graalvm.org/downloads/ and b) downloads at https://github.com/graalvm/graalvm-ce-builds/releases

### ISSUE THREAD

**Issue by rubyFeedback** [oracle/graal#6932]
Hey GraalVM folks,

This is a request for clarification in the documentation.

I read the blog entry here:

https://medium.com/graalvm/whats-new-in-graalvm-languages-161527df3d76

And found Oracle GraalVM.

However had, even after reading it, I am not entirely certain of the differences
between the following two URLs (also mentioned in the title of this issue
request):

https://www.graalvm.org/downloads/

and

https://github.com/graalvm/graalvm-ce-builds/releases

I am actually using the second URL (https://github.com/graalvm/graalvm-ce-builds/releases),
and it also replaced the older openjdk download link I used. So as far as I am concerned I
am happy with the second URL.

Now, right now I do not know whether Oracle GraalVM is different to the second URL.

So my first request is: could you folks clarify this somewhere, what the differences are
between these two download links? Or perhaps they are the same - right now I really
don't know. I assume they are not the same, but in this case I believe it may be useful
to be able to link to some article or paragraph on a website that briefly explains the
differences. This does not have to be long, a single paragraph of medium length should
suffice, just pointing at the key differences.

Personally as I do not know any differences (or not), I will continue to use 
https://github.com/graalvm/graalvm-ce-builds/releases, but perhaps in the long run
you guys can consider unifying ALL download links onto one page. That way people
may be less confused about differences. (I could test to see for any differences, but
I am perfectly happy with https://github.com/graalvm/graalvm-ce-builds/releases -
it's just that I think it would be best to be able to unify all downloads onto one page,
or at the least one NAVIGATIONAL page, so the one at https://www.graalvm.org/downloads/
which uses the corporation CSS theme, I think, could then also be used to jump towards
the github page. I am fine using either webpage, but I would like to know if the actual
downloads are then the same or different. The homepage at graalvm.org/downloads
also offers the "Download GraalVM automatically in scripts by using download URLs"
functionality, which I am sure is very useful for many folks; for me, personally, I 
actually do the oldschool manual way of just downloading the releases I need from
the github-page, both for Linux and Windows. Linux is my primary operating system
though, I do almost all java-writing on Linux.)

TL;DR of this issue request: **please consider clarifying via some documentation what
the differences are between the two mentioned URLs in the title of this issue request**.

Thanks for reading and possibly considering!

**Comment by rubyFeedback** [comment]
(Not sure why this is labeled via "bug" automatically. Perhaps I picked the wrong template - sorry for that. It is **not** a bug.)

**Comment by oubidar-Abderrahim** [comment]
Hi @rubyFeedback , 

- You can use https://www.graalvm.org/downloads/ to download Oracle GraalVM (formally known as Oracle GraalVM Enterprise Edition) which is now free to use in production and free to redistribute, at no cost, under the [GraalVM Free Terms and Conditions](https://www.oracle.com/downloads/licenses/graal-free-license.html)
- You can use https://github.com/graalvm/graalvm-ce-builds/releases to download GraalVM Community (formally known as Oracle GraalVM Community Edition). GraalVM Community Edition is distributed under [version 2 of the GNU General Public License with the “Classpath” Exception](https://github.com/oracle/graal/blob/master/LICENSE). We also recommend checking the [licenses](https://github.com/oracle/graal#license) of the individual GraalVM components (which are generally derivative of the license of a particular language).

Hope that clarifies things
